app/utils/remote_signature.py

```
@@ -28,19 +41,43 @@ def build_message_to_sign(
     timestamp: str,
     fingerprint: str,
     nonce: str,
+    headers: Dict[str, str] = None,
```

Ruff (0.15.1)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)